### DISCUSSION TITLE
How to upgrade to v1.27.0?

### DISCUSSION THREAD

**Discussion by aiden-cat** [teslamate-org/teslamate#2783] [General]
I follow the commands for docker install but this time it doesn't seem to refresh teslamate to v1.27 and is stuck on v1.25 - see below, what can I do to force upgrade to 27?

![image](https://user-images.githubusercontent.com/63931544/179872765-e45d5b75-2d1b-4087-b9e7-8bddbaabf15b.png)

**Comment by cwanja** [comment]
What does your YML file say for TeslaMate image?

**Reply by aiden-cat** [reply]
`version: "3"

services:
  teslamate:
    image: teslamate/teslamate:1.25.0
    restart: always
    environment:
      - DATABASE_USER=teslamate
      - DATABASE_PASS=secret
      - DATABASE_NAME=teslamate
      - DATABASE_HOST=database
      - MQTT_HOST=mosquitto
    ports:
      - 4000:4000
    volumes:
      - ./import:/opt/app/import
    cap_drop:
      - all

  database:
    image: postgres:12
    restart: always
    environment:
      - POSTGRES_USER=teslamate
      - POSTGRES_PASSWORD=secret
      - POSTGRES_DB=teslamate
    volumes:
      - teslamate-db:/var/lib/postgresql/data

  grafana:
    image: teslamate/grafana:latest
    restart: always
    environment:
      - DATABASE_USER=teslamate
      - DATABASE_PASS=secret
      - DATABASE_NAME=teslamate
      - DATABASE_HOST=database
    ports:
      - 3000:3000
    volumes:
      - teslamate-grafana-data:/var/lib/grafana

  mosquitto:
    image: eclipse-mosquitto:1.6
    restart: always
    ports:
      - 1883:1883
    volumes:
      - mosquitto-conf:/mosquitto/config
      - mosquitto-data:/mosquitto/data

volumes:
  teslamate-db:
  teslamate-grafana-data:
  mosquitto-conf:
  mosquitto-data:
`

**Reply by cwanja** [reply]
Change `image: teslamate/teslamate:1.25.0` to `image: teslamate/teslamate:latest` and rerun. However, if you were keeping on 1.25 for any specific reason, you will want to address that. Specifically #2302 for Pi users.

**Reply by aiden-cat** [reply]
It worked!  And yea I kept on 1.25 because of those issues way back.... no issue yet so we'll see!
